dev-tex/harvard/harvard-2.0.5-r1.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit latex-package
+
+DESCRIPTION="The harvard family of bibliographic styles"
+HOMEPAGE="https://www.ctan.org/pkg/harvard"
+# downloaded from http://mirrors.ctan.org/macros/latex/contrib/harvard.zip
+SRC_URI="mirror://gentoo/${P}.tar.bz2"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ppc x86"
```

Comment:
When performing a revbump all keywords have to be reverted to testing.  In this particular case that means
```
KEYWORDS="amd64 ppc x86"
```
should become
```
KEYWORDS="~amd64 ~ppc ~x86"
```